AgentDiscoveries-Frontend/app/src/components/search-forms/search-result.jsx

```diff
@@ -24,6 +25,12 @@ export default class SearchResult extends React.Component {
 
     renderResultBody(result) {
         return Object.keys(result).map(key => {
+
```

Comment:
Given we use the `result[key]` answer so often, could we extract it and name it to make it clearer?

e.g. 
```
const value = result[key]

if(moment(value, moment.ISO_8601, true).isValid()){
   ...
}
...
```

Reply:
In fact, this might be a nice opportunity to use a ternary & a well named function to make this more readable!

What if we wrote...

```
const value = result[key]

const display = moment(value, moment.ISO_8601, true).isValid() ?
    formatAsDate(value) :
    key.toString() + ': ' + value;

return <p key={key} id={key}>{display}</p>;
```

Reply:
Which would let us write the formatAsDate function elsewhere to use the moment.tz.guess() you suggested 😄

Reply:
I've pushed the changes and named the commit "Made displayLocalTime code neater"

thanks